### PR TITLE
ftxdiff: add page

### PR DESCRIPTION
**Pull request by GingerKiwi**
from page request: macOS Font Tools aka. Apple Font Tool Suite #8478 
Note: I'm a new dev, but experienced writer. This is my first docs PR. Feedback appreciated.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/ftxdiff.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# ftxdiff
+
+> Compares the differences between two fonts. More information can be found here https://developer.apple.com/fonts/ .
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:3]:
```suggestion
> Compare differences between two fonts. 
> More information:<https://developer.apple.com/fonts>.
```

File: pages/osx/ftxdiff.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Output differences to a text file:
+
+`ftxdiff --output FontDiff.txt FirstFont.ttc SecondFont.ttc`
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:7]:
```suggestion
`ftxdiff --output {{FontDiff.txt}} {{FirstFont.ttc}} {{SecondFont.ttc}}`
```

File: pages/osx/ftxdiff.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- Include glyph names in output:
+
+`ftxdiff --include-glyph-names` 
+
+- Include unicode names in output:
+
+`ftxdiff --include-unicode-names`
+
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:16]:
```suggestion
```

File: pages/osx/ftxdiff.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+...
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:17]:
```suggestion
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/osx/ftxdiff.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# ftxdiff
+
+> Compare differences between two fonts. 
+> More information:<https://developer.apple.com/fonts>.
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:4]:
```suggestion
> More information: <https://developer.apple.com/fonts>.
```

File: pages/osx/ftxdiff.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# ftxdiff
+
+> Compare differences between two fonts.
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:3]:
```suggestion
> Compare differences between two fonts.
```

File: pages/osx/ftxdiff.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+> Compare differences between two fonts. 
+> More information:<https://developer.apple.com/fonts>.
+
+- Output differences to a text file:
+
+`ftxdiff --output {{FontDiff.txt}} {{FirstFont.ttc}} {{SecondFont.ttc}}`
+
+- Include glyph names in output:
+
+`ftxdiff --include-glyph-names`
```
Comment (kbdharun) [inline pages/osx/ftxdiff.md:12]:
```suggestion
`ftxdiff --include-glyph-names`
```

**Review by kbdharun — COMMENTED** [review]
These suggestions would fix the Linter errors.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/osx/ftxdiff.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# ftxdiff
+
+> Compare differences between two fonts.
+> More information: <https://developer.apple.com/fonts>.
+
+- Output differences to a text file:
+
+`ftxdiff --output {{FontDiff.txt}} {{FirstFont.ttc}} {{SecondFont.ttc}}`
```
Comment (blueskyson) [inline pages/osx/ftxdiff.md:8]:
```suggestion
`ftxdiff --output {{fontdiff.txt}} {{path/to/font1.ttc}} {{path/to/font2.ttc}}`
```

File: pages/osx/ftxdiff.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# ftxdiff
+
+> Compare differences between two fonts.
+> More information: <https://developer.apple.com/fonts>.
+
+- Output differences to a text file:
+
+`ftxdiff --output {{fontdiff.txt}} {{path/to/font1.ttc}} {{path/to/font2.ttc}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/ftxdiff.md:8]:
```suggestion
- Output differences to a specific text file:

`ftxdiff --output {{path/to/fontdiff.txt}} {{path/to/font1.ttc}} {{path/to/font2.ttc}}`
```

- Optional fix: `a specific` may sound better.
- Mandatory fix: [path format violation](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Mandatory fixes must be applied, changes requested.
- This PR should be reviewed by `ftxdiff` users.
- Thanks for a contribution! ❤️

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**